<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDemo</class>
 <widget class="QMainWindow" name="ClientDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>温度调节</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>风速调节</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PowerOn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WindSpeed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>当前风速</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RoomTmp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="SpeedAdjust">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>低</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>室温</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>工作模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>目标温度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Fee">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PowerDown">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TargetTmp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>费用</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Mode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>房间号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RoomNo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TmpAdjust">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="up">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>down</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionsettings"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionsettings">
   <property name="text">
    <string>settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
